tests/api2/test_007_early_settings.py

```diff
@@ -0,0 +1,7 @@
+from middlewared.test.integration.utils import call
+
+
+def test_sysctl_arc_max_is_set():
+    """Middleware should have set this value early in boot phase
+    and this should return a number"""
+    assert call('sysctl.get_default_arc_max')
```

Comment:
Won't this write the file if it doesn't exist? We should be able to `filesystem.stat` it to check for existence.

Reply:
derp, fixed